v2/bind_test.go

```diff
@@ -60,14 +60,15 @@ const optionalFieldsBindRequestBody = `{"service_id":"test-service-id","plan_id"
 
 func TestBind(t *testing.T) {
 	cases := []struct {
-		name               string
-		enableAlpha        bool
-		request            *BindRequest
-		httpChecks         httpChecks
-		httpReaction       httpReaction
-		expectedResponse   *BindResponse
-		expectedErrMessage string
-		expectedErr        error
+		name                string
+		enableAlpha         bool
+		originatingIdentity string
```

Comment:
We should only send originating identity if alpha features are enabled, since this is still in 'validating through implementation' in the OSB API, see https://github.com/openservicebrokerapi/servicebroker/pull/222.